packages/uicore/src/components/Select/Select.tsx

```diff
@@ -256,6 +260,10 @@ export function Select(props: SelectProps): ReactElement {
     inputRef.current = el
   }, [])
 
+  useEffect(() => {
+    handleInputValueScroll()
+  }, [inputRef.current])
```

Comment:
Not sure if we should add ref in useEffect dependencies?
ref is not a state, so not sure if this will retrigger the useEffect

Reply:
As discussed over call, this is needed for the initial render with label overflowing